examples/instrumentation-quickstart/README.md

```diff
@@ -39,11 +39,22 @@ oauth2 flow (read more about the command [here][auth_command]):
 
 	gcloud auth application-default login
 
-Set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable with `export GOOGLE_APPLICATION_CREDENTIALS="$HOME/.config/gcloud/application_default_credentials.json"`
-or manually set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable to point to a service
-account key JSON file path.
+**NOTE: This method of authentication is not recommended for production environments.**
```

Comment:
There is a GH markdown syntax for this https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts